src/components/explore/AggregationsOverTimeGraph.svelte

```diff
@@ -172,35 +148,33 @@ $: if (referenceTextElement && referenceBackgroundElement) {
   yDomain={yDomain}
   yType={yScaleType}
   xType={xScaleType}
-  width={buildIDComparisonGraph.width
-    - (insufficientData ? buildIDComparisonGraph.insufficientDataAdjustment : 0)}
-  height={buildIDComparisonGraph.height}
-  bottom={buildIDComparisonGraph.bottom}
-  top={buildIDComparisonGraph.top}
-  left={buildIDComparisonGraph.left}
+  width={aggregationsOverTimeGraph.width
+    - (insufficientData ? aggregationsOverTimeGraph.insufficientDataAdjustment : 0)}
+  height={aggregationsOverTimeGraph.height}
+  bottom={aggregationsOverTimeGraph.bottom}
+  top={aggregationsOverTimeGraph.top}
+  left={aggregationsOverTimeGraph.left}
+  bottomBorder
+  borderColor={aggregationsOverTimeGraph.borderColor}
   bind:rollover={dgRollover}
   bind:hoverValue
   bind:xScale={xScale}
   bind:leftPlot={L}
   bind:rightPlot={R}
   bind:margins={margins}
-  right={buildIDComparisonGraph.right}
+  right={aggregationsOverTimeGraph.right}
   key={key}
   bind:dataGraphicMounted={dataGraphicMounted}
-  on:click={() => {
-    if (hovered.datum) {
-      reference = hovered.datum;
-    }
-  }}
+  on:click
```

Comment:
Is this valid? Should it be removed?

Reply:
yep, this forwards the event so the parent can have the `on:click` https://svelte.dev/tutorial/event-forwarding